src/merge-analysis/into-classes.js

```diff
@@ -76,7 +77,7 @@ function intoClasses(classes, members) {
       if (!membersLookup[path]) { 
         // Inform only if there is no member either with this path
         // Object properties containing JSDocs are processed separately
-        info(member, 'class member', `name ${member.name} @memberof parameter has not defined object path of "${path}"`);
+        error(member, 'class member', `name ${member.name} @memberof parameter has not defined object path of "${path}"`);
```

Comment:
How do I get this to occur?
I put this in and nothing triggered:
```js
	/**
	 * Figures out the position of a future marker but only if it is displayed on the screen.
	 * @param  {CIQ.Marker} marker The marker to check
	 * @memberOf CIQ2.ChartEngine2
	 */
	CIQ.ChartEngine.prototype.futureTickIfDisplayed = function (marker) {
```

Reply:
change the memberof tag to something like: `@memberOf CIQ.chartEngine`

Reply:
What's wrong with what I did (CIQ.ChartEngine2)?

Reply:
This log only works on "non class-like" type of objects where we define "class-like" as having a capitalized first letter of the name. So having a CIQ.ChartEngine2 or a member with a type in it will not get into this path.

There is no consistent way to tell when something is meant to be a namespace or interface from another module so we can't assume that if there's a member with a different class name that it is an error. I tried this yesterday with an error log and everytime you first extend CIQ.ChartEngine outside of chartiq.js it would spit an error (even though its not an error).

Reply:
That's really limiting what this can do.  So it only catches a memberof where I forgot to capitalize something?

Reply:
Yes or we need to define classLike differently (see line 57)

Reply:
Can it collate the class names from the code by searching for @namespace, @name, etc?

Reply:
We may be able to add required interfaces (due to CIQ namespace possible but not required addition) using JSDoc that would open possibility for throwing error for misspelled `@memberOf` properties.

Reply:
> Can it collate the class names from the code by searching for @namespace, @name, etc?

Checking class names is what I attempted to do but I couldn't get it to not throw an errors when it re-encountered `CIQ.ChartEngine`. Checking namespaces is not something that I could as easily do but may be possible.

Reply:
"Interfaces would be required for type references that are not contained in the currently processed file."
Can you give me some examples of where this would happen?

Reply:
`js/core/typedefs.js` definition is to solve the issue.  It probably does not solve it comprehensively for all generated files.

Reply:
@normunds-ciq are you suggesting that we do something like add a typedefs.js file to each folder so that we can define the imported types?